geode-core/src/main/java/org/apache/geode/internal/cache/InitialImageOperation.java

```diff
@@ -1077,7 +1081,8 @@ protected Set<Object> processReceivedRVV(RegionVersionVector remoteRVV,
         if (id == null) {
           id = myId;
         }
-        if (!remoteRVV.contains(id, stamp.getRegionVersion())) {
+        foundIds.add(id);
```

Comment:
This will add `id` to `foundIds`, even for the persistent case. However, according to the description of GEODE-10290, it is for the non-persistent case:
>In non-persistent but concurrent-check enabled case, members departed will be marked. They should be removed from RVV during GII to prevent memberToVersion in RVV grows bigger and bigger.

So I am not sure if we should add a `if` check here. e.g. `if (!isPersistentRegion)` .

Reply:
it won't hurt to add id into foundIds. I will keep it the same as GII provider part.